2018/12/01 08:34:49 [warn] 8#8: *107 an upstream response is buffered to a temporary file /var/cache/nginx/proxy_temp/1/00/0000000001 while reading upstream, client: 172.18.0.1, server: db.localhost, request: "GET /js/vendor/codemirror/lib/codemirror.js?v=4.8.3 HTTP/1.1", upstream: "http://172.18.0.6:80/js/vendor/codemirror/lib/codemirror.js?v=4.8.3", host: "db.localhost"
2018/12/02 18:57:33 [error] 8#8: *1 open() "/usr/share/nginx/html/favicon.ico" failed (2: No such file or directory), client: 172.18.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", host: "wtftrackr.pw"
2018/12/02 18:57:33 [error] 8#8: *1 open() "/usr/share/nginx/html/favicon.png" failed (2: No such file or directory), client: 172.18.0.1, server: localhost, request: "GET /favicon.png HTTP/1.1", host: "wtftrackr.pw"
2018/12/02 18:57:33 [error] 8#8: *1 open() "/usr/share/nginx/html/favicon.gif" failed (2: No such file or directory), client: 172.18.0.1, server: localhost, request: "GET /favicon.gif HTTP/1.1", host: "wtftrackr.pw"
2018/12/02 18:57:33 [error] 8#8: *1 open() "/usr/share/nginx/html/favicon.jpg" failed (2: No such file or directory), client: 172.18.0.1, server: localhost, request: "GET /favicon.jpg HTTP/1.1", host: "wtftrackr.pw"
2018/12/20 15:24:12 [warn] 7#7: *8 an upstream response is buffered to a temporary file /var/cache/nginx/proxy_temp/1/00/0000000001 while reading upstream, client: 172.18.0.1, server: db.localhost, request: "GET /js/vendor/codemirror/lib/codemirror.js?v=4.8.3 HTTP/1.1", upstream: "http://172.18.0.6:80/js/vendor/codemirror/lib/codemirror.js?v=4.8.3", host: "db.localhost"
2018/12/21 12:06:54 [warn] 8#8: *9 an upstream response is buffered to a temporary file /var/cache/nginx/proxy_temp/1/00/0000000001 while reading upstream, client: 172.18.0.1, server: db.localhost, request: "GET /js/vendor/jquery/jquery-ui.min.js?v=4.8.3 HTTP/1.1", upstream: "http://172.18.0.4:80/js/vendor/jquery/jquery-ui.min.js?v=4.8.3", host: "db.localhost"
2018/12/25 09:30:43 [error] 8#8: *1 open() "/usr/share/nginx/html/favicon.ico" failed (2: No such file or directory), client: 172.18.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", host: "wtftrackr.pw"
2018/12/25 09:30:44 [error] 8#8: *1 open() "/usr/share/nginx/html/favicon.png" failed (2: No such file or directory), client: 172.18.0.1, server: localhost, request: "GET /favicon.png HTTP/1.1", host: "wtftrackr.pw"
2018/12/25 09:30:44 [error] 8#8: *1 open() "/usr/share/nginx/html/favicon.gif" failed (2: No such file or directory), client: 172.18.0.1, server: localhost, request: "GET /favicon.gif HTTP/1.1", host: "wtftrackr.pw"
2018/12/25 09:30:44 [error] 8#8: *1 open() "/usr/share/nginx/html/favicon.jpg" failed (2: No such file or directory), client: 172.18.0.1, server: localhost, request: "GET /favicon.jpg HTTP/1.1", host: "wtftrackr.pw"
2018/12/25 22:42:43 [error] 28#28: *2 open() "/usr/share/nginx/html/favicon.ico" failed (2: No such file or directory), client: 172.18.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", host: "miaomiaodai.localhost", referrer: "http://miaomiaodai.localhost/"
2018/12/25 22:43:52 [error] 55#55: *5 open() "/usr/share/nginx/html/api/base" failed (2: No such file or directory), client: 172.18.0.1, server: localhost, request: "GET /api/base HTTP/1.1", host: "miaomiaodai.localhost"
2018/12/25 22:45:49 [error] 81#81: *8 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://127.0.0.1:8000/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:45:49 [warn] 81#81: *8 upstream server temporarily disabled while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://127.0.0.1:8000/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:45:49 [error] 81#81: *8 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://127.0.0.1:8000/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:45:49 [warn] 81#81: *8 upstream server temporarily disabled while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://127.0.0.1:8000/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:45:50 [error] 81#81: *8 no live upstreams while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://localhost/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:45:51 [error] 81#81: *8 no live upstreams while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://localhost/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:45:52 [error] 81#81: *8 no live upstreams while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://localhost/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:48:37 [error] 81#81: *11 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://127.0.0.1:8000/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:48:37 [warn] 81#81: *11 upstream server temporarily disabled while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://127.0.0.1:8000/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:48:37 [error] 81#81: *11 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://127.0.0.1:8000/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:48:37 [warn] 81#81: *11 upstream server temporarily disabled while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://127.0.0.1:8000/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:48:40 [error] 81#81: *11 no live upstreams while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://localhost/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:48:41 [error] 81#81: *11 no live upstreams while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://localhost/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:48:41 [error] 81#81: *11 no live upstreams while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://localhost/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:48:41 [error] 81#81: *11 no live upstreams while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://localhost/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:48:42 [error] 81#81: *11 no live upstreams while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://localhost/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:48:42 [error] 81#81: *11 no live upstreams while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://localhost/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:48:54 [error] 81#81: *11 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://127.0.0.1:8000/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:48:54 [warn] 81#81: *11 upstream server temporarily disabled while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://127.0.0.1:8000/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:48:54 [error] 81#81: *11 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://127.0.0.1:8000/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:48:54 [warn] 81#81: *11 upstream server temporarily disabled while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://127.0.0.1:8000/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:48:55 [error] 81#81: *11 no live upstreams while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://localhost/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:49:09 [error] 81#81: *11 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://127.0.0.1:8000/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:49:09 [warn] 81#81: *11 upstream server temporarily disabled while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://127.0.0.1:8000/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:49:09 [error] 81#81: *11 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://127.0.0.1:8000/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:49:09 [warn] 81#81: *11 upstream server temporarily disabled while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://127.0.0.1:8000/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:49:10 [error] 81#81: *11 no live upstreams while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://localhost/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:53:32 [error] 110#110: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://127.0.0.1:8000/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:53:32 [warn] 110#110: *21 upstream server temporarily disabled while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://127.0.0.1:8000/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:53:32 [error] 110#110: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://127.0.0.1:8000/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:53:32 [warn] 110#110: *21 upstream server temporarily disabled while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://127.0.0.1:8000/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:53:33 [error] 110#110: *21 no live upstreams while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://localhost/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:53:34 [error] 110#110: *21 no live upstreams while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://localhost/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:53:38 [error] 110#110: *21 no live upstreams while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://localhost/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:53:42 [error] 110#110: *21 no live upstreams while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://localhost/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:54:11 [error] 110#110: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://127.0.0.1:8000/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:54:11 [warn] 110#110: *21 upstream server temporarily disabled while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://127.0.0.1:8000/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:54:11 [error] 110#110: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://127.0.0.1:8000/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:54:11 [warn] 110#110: *21 upstream server temporarily disabled while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://127.0.0.1:8000/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:54:14 [error] 110#110: *21 no live upstreams while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://localhost/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:55:35 [error] 110#110: *31 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://127.0.0.1:8000/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:55:35 [warn] 110#110: *31 upstream server temporarily disabled while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://127.0.0.1:8000/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:55:35 [error] 110#110: *31 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://127.0.0.1:8000/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:55:35 [warn] 110#110: *31 upstream server temporarily disabled while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://127.0.0.1:8000/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:58:31 [error] 135#135: *34 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://127.0.0.1:8000/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:58:31 [warn] 135#135: *34 upstream server temporarily disabled while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://127.0.0.1:8000/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:58:31 [error] 135#135: *34 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://127.0.0.1:8000/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:58:31 [warn] 135#135: *34 upstream server temporarily disabled while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://127.0.0.1:8000/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:58:32 [error] 135#135: *34 no live upstreams while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET /api/base HTTP/1.1", upstream: "http://localhost/api/base", host: "miaomiaodai.localhost"
2018/12/25 22:59:20 [error] 135#135: *34 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:8000/", host: "miaomiaodai.localhost"
2018/12/25 22:59:20 [warn] 135#135: *34 upstream server temporarily disabled while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:8000/", host: "miaomiaodai.localhost"
2018/12/25 22:59:20 [error] 135#135: *34 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:8000/", host: "miaomiaodai.localhost"
2018/12/25 22:59:20 [warn] 135#135: *34 upstream server temporarily disabled while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:8000/", host: "miaomiaodai.localhost"
2018/12/25 22:59:27 [error] 135#135: *42 open() "/usr/share/nginx/html/favicon.ico" failed (2: No such file or directory), client: 172.18.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", host: "miaomiaodai.local", referrer: "http://miaomiaodai.local/"
2018/12/25 22:59:46 [error] 163#163: *44 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.local, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:8000/", host: "miaomiaodai.local"
2018/12/25 22:59:46 [warn] 163#163: *44 upstream server temporarily disabled while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.local, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:8000/", host: "miaomiaodai.local"
2018/12/25 22:59:46 [error] 163#163: *44 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.local, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:8000/", host: "miaomiaodai.local"
2018/12/25 22:59:46 [warn] 163#163: *44 upstream server temporarily disabled while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.local, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:8000/", host: "miaomiaodai.local"
2018/12/25 22:59:52 [error] 163#163: *44 no live upstreams while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.local, request: "GET / HTTP/1.1", upstream: "http://localhost/", host: "miaomiaodai.local"
2018/12/25 23:00:48 [error] 189#189: *48 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.local, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:8000/", host: "miaomiaodai.local"
2018/12/25 23:00:51 [error] 189#189: *48 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.local, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:8000/", host: "miaomiaodai.local"
2018/12/25 23:00:52 [error] 189#189: *48 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.local, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:8000/", host: "miaomiaodai.local"
2018/12/25 23:01:15 [error] 189#189: *48 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.local, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:8000/", host: "miaomiaodai.local"
2018/12/25 23:01:16 [error] 189#189: *48 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.local, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:8000/", host: "miaomiaodai.local"
2018/12/25 23:01:17 [error] 189#189: *48 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.local, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:8000/", host: "miaomiaodai.local"
2018/12/25 23:01:17 [error] 189#189: *48 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.local, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:8000/", host: "miaomiaodai.local"
2018/12/25 23:01:17 [error] 189#189: *48 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.local, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:8000/", host: "miaomiaodai.local"
2018/12/25 23:01:18 [error] 189#189: *48 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.local, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:8000/", host: "miaomiaodai.local"
2018/12/25 23:01:26 [error] 189#189: *48 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.local, request: "GET /admin/ HTTP/1.1", upstream: "http://127.0.0.1:8000/admin/", host: "miaomiaodai.local"
2018/12/25 23:02:01 [error] 189#189: *48 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.local, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:8000/", host: "miaomiaodai.local"
2018/12/25 23:02:20 [error] 216#216: *65 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:8080/", host: "miaomiaodai.localhost"
2018/12/25 23:02:22 [error] 216#216: *65 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: miaomiaodai.localhost, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:8080/", host: "miaomiaodai.localhost"
2019/01/06 09:10:13 [error] 93#93: *1 open() "/usr/share/nginx/html/favicon.ico" failed (2: No such file or directory), client: 172.18.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2019/01/06 09:23:44 [error] 34#34: *4 user "lizs": password mismatch, client: 172.18.0.1, server: auth.localhost, request: "GET / HTTP/1.1", host: "auth.localhost"
2019/01/06 09:25:13 [error] 34#34: *4 open() "/usr/share/nginx/html/favicon.ico" failed (2: No such file or directory), client: 172.18.0.1, server: auth.localhost, request: "GET /favicon.ico HTTP/1.1", host: "auth.localhost", referrer: "http://auth.localhost/"
2019/01/06 09:31:24 [error] 64#64: *10 open() "/etc/nginx/htpasswd" failed (2: No such file or directory), client: 172.18.0.1, server: auth.localhost, request: "GET / HTTP/1.1", host: "auth.localhost"
2019/01/06 09:31:27 [error] 64#64: *10 open() "/etc/nginx/htpasswd" failed (2: No such file or directory), client: 172.18.0.1, server: auth.localhost, request: "GET / HTTP/1.1", host: "auth.localhost"
2019/01/06 09:31:29 [error] 64#64: *10 open() "/etc/nginx/htpasswd" failed (2: No such file or directory), client: 172.18.0.1, server: auth.localhost, request: "GET / HTTP/1.1", host: "auth.localhost"
2019/01/06 09:32:02 [error] 119#119: *13 open() "/etc/nginx/htpasswd" failed (2: No such file or directory), client: 172.18.0.1, server: auth.localhost, request: "GET / HTTP/1.1", host: "auth.localhost"
2019/01/06 09:32:24 [error] 119#119: *13 open() "/etc/nginx/htpasswd" failed (2: No such file or directory), client: 172.18.0.1, server: auth.localhost, request: "GET / HTTP/1.1", host: "auth.localhost"
2019/01/10 17:33:26 [warn] 8#8: *5 an upstream response is buffered to a temporary file /var/cache/nginx/proxy_temp/1/00/0000000001 while reading upstream, client: 172.18.0.1, server: db.localhost, request: "GET /js/vendor/jquery/jquery-ui.min.js?v=4.8.3 HTTP/1.1", upstream: "http://172.18.0.5:80/js/vendor/jquery/jquery-ui.min.js?v=4.8.3", host: "db.localhost"
2019/01/10 17:33:26 [warn] 8#8: *2 an upstream response is buffered to a temporary file /var/cache/nginx/proxy_temp/2/00/0000000002 while reading upstream, client: 172.18.0.1, server: db.localhost, request: "GET /js/functions.js?v=4.8.3 HTTP/1.1", upstream: "http://172.18.0.5:80/js/functions.js?v=4.8.3", host: "db.localhost"
2019/01/12 08:28:57 [warn] 8#8: *220 an upstream response is buffered to a temporary file /var/cache/nginx/proxy_temp/1/00/0000000001 while reading upstream, client: 172.18.0.1, server: db.localhost, request: "GET /js/functions.js?v=4.8.3 HTTP/1.1", upstream: "http://172.18.0.5:80/js/functions.js?v=4.8.3", host: "db.localhost"
2019/01/12 16:39:11 [warn] 8#8: *499 an upstream response is buffered to a temporary file /var/cache/nginx/proxy_temp/2/00/0000000002 while reading upstream, client: 172.18.0.1, server: db.localhost, request: "GET /js/vendor/codemirror/lib/codemirror.js?v=4.8.3 HTTP/1.1", upstream: "http://172.18.0.5:80/js/vendor/codemirror/lib/codemirror.js?v=4.8.3", host: "db.localhost"
2019/01/17 06:41:21 [emerg] 1#1: host not found in upstream "miaomiaodai" in /etc/nginx/conf.d/miaomiaodai.conf:7
2019/01/17 07:03:37 [emerg] 1#1: host not found in upstream "miaomiaodai" in /etc/nginx/conf.d/miaomiaodai.conf:7
2019/01/17 07:27:05 [emerg] 1#1: host not found in upstream "miaomiaodai" in /etc/nginx/conf.d/miaomiaodai.conf:7
2019/01/17 15:18:49 [emerg] 1#1: host not found in upstream "miaomiaodai" in /etc/nginx/conf.d/miaomiaodai.conf:7
